ios/ServicexSdkRn.swift

```diff
@@ -62,13 +71,25 @@ class ServicexSdkRn: NSObject {
         }
         
         print(_productTypes)
-        CredifyServiceX.offer.getOffers(phoneNumber: user?.phoneNumber,
-                                                      countryCode: user?.countryCode,
-                                                      localId: user!.id,
-                                                      credifyId: user?.credifyId, productTypes:_productTypes) { [weak self] result in
-            switch result {
-            case .success(let offers):
-                self?.listOffer = offers
+        
```

Comment:
@danhnn `let user = self.parseUserProfile(value: userInput!)` should use `guard` to avoid from run time crash